azerothcore/azerothcore-wotlk
# fix(Scripts/Brewfest): Reload faction auras for revelers after respawn.

Fixes #7871
**#9263 is a requirement.**



## Issues Addressed:

- Closes #7871

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

.event start 24
Go to Brewfest locations infront of Orgrimmar or Ironforge
Observe the revelers as ally and horde char at both locations


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
